libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateErrors.cs

```diff
@@ -62,12 +62,16 @@ public class TemplateErrors
 
         public const string LoopDetected = "Loop detected:";
 
-        public const string SyntaxError = "Unexpected content. Expecting either a comment or a template definition or an import statement.";
+        public const string SyntaxError = "Unexpected content. Expecting either a comment or a template definition or an import statement or an option definition.";
 
         public const string InvalidMemory = "Scope is not a LG customized memory.";
 
         public const string StaticFailure = "Static failure with the following error.";
 
+        public const string ImportFail = "Import should be placed at the beginning of the file.";
+
+        public const string OptionFail = "Option should be placed at the beginning of the file.";
```

Comment:
Is there an order to these?  I suspect the rule is that both should be put before any template files, i.e. "Import must be before the first template defintion."  Same for option.